reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -321,278 +310,308 @@ class StaticHtmlReporter : Reporter() {
         return outputFile
     }
 
-    private fun createContent(tabularScanRecord: ReportTableModel) =
-            buildString {
-                if (tabularScanRecord.metadata.isNotEmpty()) {
-                    append("<h2>Metadata</h2>")
-                    append("<table class=\"ort-report-metadata\"><tbody>")
-                    tabularScanRecord.metadata.forEach { (key, value) ->
-                        append("""
-                        <tr>
-                            <td>$key</td>
-                            <td>${if (value.isValidUrl()) "<a href=\"$value\">$value</a>" else value}</td>
-                        </tr>
-                        """.trimIndent())
+    private fun renderHtml(reportTableModel: ReportTableModel): String {
+        val document = DocumentBuilderFactory.newInstance().newDocumentBuilder().newDocument()
+
+        document.append.html {
+            lang = "en"
+
+            head {
+                meta(charset = "UTF-8")
```

Comment:
The new code seems to automatically add

    <META http-equiv="Content-Type" content="text/html; charset=UTF-8">

(with upper case "META"!) so I guess we could drop this line in exchange.

Reply:
Also see https://github.com/Kotlin/kotlinx.html/issues/63.

Reply:
Done. I guess it could be fixed by setting another `TransformerFactory` as suggested in the ticket, but it's not that important.